SPECS-EXTENDED/xdp-tools/xdp-tools.spec

```
@@ -0,0 +1,161 @@
+Summary:        Utilities and example programs for use with XDP
+Name:           xdp-tools
+Version:        1.2.9
+Release:        1%{?dist}
+License:        GPL-2.0-only AND LGPL-2.1-only AND BSD-2-Clause
+URL:            https://github.com/xdp-project/%{name}
+Source0:        https://github.com/xdp-project/%{name}/releases/download/v%{version}/xdp-tools-%{version}.tar.gz
+%global _soversion 1.2.0
+# added to prevent stripping of XDP objects, which Mariner binutils doesn't
+#  yet seem to work with. this ought be removed when possible FIXME.
+%global _enable_debug_package 0
+# find-debuginfo produces empty debugsourcefiles.list
+# disable the debug package to avoid rpmbuild error'ing out because of this
+%global debug_package %{nil}
+%global __os_install_post %{_lib}/rpm/brp-compress %{nil}
```

Still getting a build failure, due to `%{_lib}/rpm/brp-compress` expanding to `lib/rpm/brp-compress`, which doesn't exist in the context of the working directory here.

This doesn't quite work with [the way we structure our post-install scripts](https://github.com/microsoft/CBL-Mariner/blob/main/SPECS/mariner-rpm-macros/macros#L72-L87)- changing `%__os_install_post` like this is taking an axe to something that just needs a chisel. Defining `%{__strip}` in the following way should fix the issue:

```suggestion
%global __strip /bin/true
```

---

cheerfully applied, thanks! odd, i wonder why this is working in our mariner build.